### ISSUE TITLE
Error after latest update - Missing Speed Sensors

### ISSUE THREAD

**Issue by NiiXXiiN** [Rem0o/FanControl.Releases#1010]
Missing Speed Sensors occurring only after latest update.
I tried to increase startup delay up to 2 minutes with no luck.
I suppose something was changed in the latest update that bugged the sensor discovery as i see multiple related bug reports.

There were no log.txt relevant entries.

[cpu-id.txt](https://github.com/Rem0o/FanControl.Releases/files/9329292/cpu-id.txt)

![2022-08-12 (1)](https://user-images.githubusercontent.com/76655871/184426055-757885a5-5e5a-4726-bce8-fa3f9be33b14.png)

**Comment by Rem0o** [comment]
If refreshed, do they come back up?

**Comment by NiiXXiiN** [comment]
After 2 or 3 attempts to refresh they do. Even even if shutdown the app and restart it it has the same behavior.

**Comment by powpow-git** [comment]
I started having this issue as well since update 1.24. Only happens for my 3060 GPU where thee is a question mark at the bottom of the two fans for it saying "this control is being overridden by an external source" when moused over and the fans do not come on at all and causing my temps to go up to 92 degrees c and card starts to throttle down to not overheat. 

I have to do a refresh sensors detection for it to turn on my GPU fans and start using the curves but when I close the game I am playing and launch again the fans do not come on and the question mark appears again after a while and I have to refresh again to get it working. Something broke the sensor detect since last update as I did not have this issue at all with prior updates.
